topics/persistence.md

```diff
@@ -203,6 +222,25 @@ corruption happened to be in the initial part of the file.
 Log rewriting uses the same copy-on-write trick already in use for
 snapshotting.  This is how it works:
 
+**Redis >= 7.0**
+
+* Redis [forks](http://linux.die.net/man/2/fork), so now we have a child
+and a parent process.
+
+* The child starts writing the new AOF in a temporary file.
+
+* The parent opens a new increments AOF file to continue writing updates.
+  If the rewriting fails, the old base and increments files (if we have) plus this newly opened increments file represent the complete updated dataset,
+  so we are safe.
```

Comment:
"increments file" sounds odd, i'd refer "incremental file"
but i see you also have "increments files", could this be "incremental files"?
```suggestion
* The parent opens a new increments AOF file to continue writing updates.
  If the rewriting fails, the old base and increments files (if there are any) plus this newly opened increments file represent the complete updated dataset,
  so we are safe.
```

Reply:
> "increments file" sounds odd, i'd refer "incremental file"

I think the best is "increment files", i'll fix